main.nf

```diff
@@ -539,6 +552,12 @@ else log.info "\033[2m----------------------------------------------------\033[0
 
 if ('mutect2' in tools && !(params.pon)) log.warn "[nf-core/sarek] Mutect2 was requested, but as no panel of normals were given, results will not be optimal"
 
+
+// Define regular variables so that they can be overwritten
+clip_r1 = params.clip_r1
+clip_r2 = params.clip_r2
+three_prime_clip_r1 = params.three_prime_clip_r1
+three_prime_clip_r2 = params.three_prime_clip_r2
```

Comment:
Remove these lines. Im assuming you got this from the `nf-core/rnaseq` pipeline. These would only need to be initialised if you want to overwrite them before providing them to the `TrimGalore` process. They needed to be overwritten in the `nf-core/rnaseq` pipeline for a particular type of RNA-seq protocol i.e. `params.pico`:
https://github.com/nf-core/rnaseq/blob/3b6df9bd104927298fcdf69e97cca7ff1f80527c/main.nf#L145